laa_court_data_api_app/routers/defendants.py

```diff
@@ -33,6 +34,14 @@ async def get_defendants(urn: Optional[str] = None,
         logging.info(f"Defendants_Get_Urn_{urn}")
         cda_response = await client.get("/api/internal/v2/prosecution_cases",
                                         params={"filter[prosecution_case_reference]": urn})
+    elif asn:
+        logging.info(f"Defendants_Get_asn_{asn}")
```

Comment:
I'ma be really annoying and ask that 'asn' and 'nino' be pascal case just for predictability when searching through logs